src/components/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -32,18 +31,24 @@ const LinearProgressBar: React.FC<LinearProgressBarProps> = (props) => {
 	return (
 		<>
 			{props.value > 0 ? (
-				<div className={`relative w-100 h-3 br-10 overflow-x-hidden ${props.className}`} style={props.style}>
-					<div className={`absolute w-100 h-3 br-10 bg-43`} style={{ background: `${props.backgroundColor}` }} />
+				<div className={`relative h-3 w-full overflow-x-hidden rounded-xl ${props.className}`} style={props.style}>
```

Comment:
@manishdoley23 can we not use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress for creating determinant progress bar? try this

Reply:
the problem with the default progress bar is the color of the progress sent is not customizable